addons/hr_gamification/wizard/gamification_badge_user_wizard.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+
+
+class GamificationBadgeUserWizard(models.TransientModel):
+    _name = 'gamification.badge.user.wizard'
+    _inherit = ['gamification.badge.user.wizard']
+
+    employee_id = fields.Many2one('hr.employee', string='Employee', required=True)
+    user_id = fields.Many2one('res.users', string='User', related='employee_id.user_id', store=True)
+
+    @api.one
```

Comment:
`@api.one` is generally a bad idea, but it seems like an even worse one for action buttons.